rego/rego.go

```diff
@@ -1800,7 +1809,8 @@ func (r *Rego) partial(ctx context.Context, ectx *EvalContext) (*PartialQueries,
 		WithRuntime(r.runtime).
 		WithIndexing(ectx.indexing).
 		WithPartialNamespace(ectx.partialNamespace).
-		WithSkipPartialNamespace(r.skipPartialNamespace)
+		WithSkipPartialNamespace(r.skipPartialNamespace).
```

Comment:
We might want to add a test for the Rego package that hits this, just to ensure the plumbing is setup correctly.

Reply:
Added a test case.